resources/lang/en/tables.php

```diff
@@ -148,6 +148,7 @@
             'to'                     => 'To',
             'from'                   => 'From',
             'transfers'              => 'Transfers',
+            'batch_transfers'        => 'Batch Transfers',
```

Comment:
maybe we can go back to multipayment for it now that the contract is updated and called that

Reply:
Updated - but do we have an "official" terminology for it? E.g. "MultiPayment", "Multipayment", "Multi-Payment" or "Multi Payment" :thinking: